packages/cli/src/commands/generate/scaffold/__tests__/__snapshots__/scaffold.test.js.snap

```diff
@@ -273,263 +273,9 @@ export default PostForm
-        nested: {
-          foobar: 'I have no imagination',
-          two: 'Second nested item',
-        },
-        five: 5,
-        bool: false,
-      })
-    ).toMatch(/.+\\n.+\\w\\.\\.\\.$/s)
-  })
-})
-
-describe('timeTag', () => {
-  it('renders a date', async () => {
-    render(<div>{timeTag(new Date('1970-08-20').toUTCString())}</div>)
-
-    await waitFor(() => screen.getByText(/1970.*00:00:00/))
-  })
-
-  it('can take an empty input string', async () => {
-    expect(timeTag('')).toEqual('')
-  })
-})
-
-describe('jsonDisplay', () => {
-  it('produces the correct output', () => {
-    expect(
-      jsonDisplay({
-        title: 'TOML Example (but in JSON)',
-        database: {
-          data: [['delta', 'phi'], [3.14]],
-          enabled: true,
-          ports: [8000, 8001, 8002],
-          temp_targets: {
-            case: 72.0,
-            cpu: 79.5,
-          },
-        },
-        owner: {
-          dob: '1979-05-27T07:32:00-08:00',
-          name: 'Tom Preston-Werner',
-        },
-        servers: {
-          alpha: {
-            ip: '10.0.0.1',
-            role: 'frontend',
-          },
-          beta: {
-            ip: '10.0.0.2',
-            role: 'backend',
-          },
-        },
-      })
-    ).toMatchInlineSnapshot(\`
-      <pre>
-        <code>
-          {
-        "title": "TOML Example (but in JSON)",
-        "database": {
-          "data": [
-            [
-              "delta",
-              "phi"
-            ],
-            [
-              3.14
-            ]
-          ],
-          "enabled": true,
-          "ports": [
-            8000,
-            8001,
-            8002
-          ],
-          "temp_targets": {
-            "case": 72,
-            "cpu": 79.5
-          }
-        },
-        "owner": {
-          "dob": "1979-05-27T07:32:00-08:00",
-          "name": "Tom Preston-Werner"
-        },
-        "servers": {
-          "alpha": {
-            "ip": "10.0.0.1",
-            "role": "frontend"
-          },
-          "beta": {
-            "ip": "10.0.0.2",
-            "role": "backend"
-          }
-        }
-      }
-        </code>
-      </pre>
-    \`)
-  })
-})
+exports[`in javascript (default) mode creates a formatters function file 1`] = `undefined`;
 
-describe('checkboxInputTag', () => {
-  it('can be checked', () => {
-    render(checkboxInputTag(true))
-    expect(screen.getByRole('checkbox')).toBeChecked()
-  })
-
-  it('can be unchecked', () => {
-    render(checkboxInputTag(false))
-    expect(screen.getByRole('checkbox')).not.toBeChecked()
-  })
-
-  it('is disabled when checked', () => {
-    render(checkboxInputTag(true))
-    expect(screen.getByRole('checkbox')).toBeDisabled()
-  })
-
-  it('is disabled when unchecked', () => {
-    render(checkboxInputTag(false))
-    expect(screen.getByRole('checkbox')).toBeDisabled()
-  })
-})
-"
-`;
+exports[`in javascript (default) mode creates a formatters function test file 1`] = `undefined`;
```

Comment:
Here too!

Reply:
(this could indicate the scaffold generator isn't working as expected too)

Reply:
Same as above